spec/services/dossier_filter_service_spec.rb

```diff
@@ -273,6 +273,23 @@ def to_filter((label, filter)) = FilteredColumn.new(column: procedure.find_colum
       it { is_expected.to eq([dossier_no, dossier_yes].map(&:id)) }
     end
 
+    context 'for labels table' do
+      let(:column) { procedure.find_column(label: 'Labels') }
+      let(:order) { 'asc' }
+
+      let(:label_a) { Label.create(name: "a", color: 'green-bourgeon', procedure:) }
+      let(:label_z) { Label.create(name: "z", color: 'green-bourgeon', procedure:) }
+      let!(:dossier_a) { create(:dossier, procedure:) }
+      let!(:dossier_z) { create(:dossier, procedure:) }
```

Comment:
Tu pourrais nous en créer 3 dans un ordre 'désordonné' pour être sur que le tri fonctionne ? (la j'ai comme un doute que `dossier_label_a` soit appelé.

```suggestion
      let!(:dossier_z) { create(:dossier, procedure:) }
      let!(:dossier_a) { create(:dossier, procedure:) }
      let!(:dossier_b) { create(:dossier, procedure:) }
```

Reply:
en effet c'était pas appelé et en plus l'ordre n'etait pas bon - ça triait par id…